### ISSUE TITLE
Texture replacement PPSSPP Yugioh GX

### ISSUE THREAD

**Issue by peachy47** [hrydgard/ppsspp#17060]
### Game or games this happens in

ULUS-10136 - Yu-Gi-Oh GX Tag Force

### What area of the game

Any area of the game.. problem with loading textures (hd pack)
is there a cache problem or something like that? PPSSPP RAM related maybe.
note: this error continues until restarting the game, it is not a slutter that resolves itself with time.

### What happens

after a while the cards become incorrect colors, monsters get magic colors and textures in general start to get black and messy.. 
![image](https://user-images.githubusercontent.com/127104757/223101122-59108dcb-4503-4389-8fd0-a683cc2c356b.png)

### What should happen

All cards images are in hd after loading which is usually fast.

### Platform

Windows

### Mobile phone model or graphics card

NVIDIA GeForce 540M

### PPSSPP version affected

v1.14.4

### Graphics backend (3D API)

Direct3D 11

### Checklist

- [X] Test in the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already fixed.
- [X] [Search for other reports](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same issue.
- [ ] Try resetting settings or older versions and include if the issue is related.
- [ ] Try changing graphics settings to determine if one causes the glitch (especially speed hacks or enhancements/replacements.)
- [ ] Include logs or screenshots of issue.

**Comment by Gamemulatorer** [comment]
Dup https://github.com/hrydgard/ppsspp/issues/16966

**Comment by hrydgard** [comment]
Yeah almost certainly the same problem. I'll be working on the texture replacement stuff a bit soon.

**Comment by unknownbrackets** [comment]
Have you tried the latest git build?  There was an issue that was fixed in the stable version you indicated you're using.

-[Unknown]

**Comment by peachy47** [comment]
> Have you tried the latest git build? There was an issue that was fixed in the stable version you indicated you're using.
> 
> -[Unknown]

I'm using v1.13 now. It works perfectly but I'll keep an eye out for future updates.

**Comment by Gamemulatorer** [comment]
> > Have you tried the latest git build? There was an issue that was fixed in the stable version you indicated you're using.
> > -[Unknown]
> 
> I'm using v1.13 now. It works perfectly but I'll keep an eye out for future updates.

Try the recently build apk
https://buildbot.orphis.net/ppsspp/index.php?m=fulllist
